src/main/scala/edu/ie3/powerFactory2psdm/converter/SubnetBuilder.scala

```diff
@@ -31,19 +32,19 @@ object SubnetBuilder extends LazyLogging {
     * Takes the grid graph and builds up the different [[Subnet]]s from it.
     *
     * @param gridGraph the built grid graph represented by the UUIDS of the nodes and the lines connecting them
-    * @param uuid2Node the mapping between UUID and the corresponding node
+    * @param id2Node the mapping between UUID and the corresponding node
     * @return the list of all subnets of the grid
     */
   def buildSubnets(
-      gridGraph: Multigraph[UUID, DefaultEdge],
-      uuid2Node: Map[UUID, Nodes]
+      gridGraph: Multigraph[String, DefaultEdge],
```

Comment:
In conjunction with my proposal for `GridGraphBuilder`

```suggestion
      gridGraph: AsUnmodifiableGraph[String, DefaultEdge],
```